yarn.lock

```diff
@@ -2415,6 +2418,11 @@
   resolved "https://registry.yarnpkg.com/@react-native-picker/picker/-/picker-2.6.1.tgz#3b20ddd1385fab0487db103dc6519570f8892e6d"
   integrity sha512-oJftvmLOj6Y6/bF4kPcK6L83yNBALGmqNYugf94BzP0FQGpHBwimVN2ygqkQ2Sn2ZU3pGUZMs0jV6+Gku2GyYg==
 
+"@react-native/assets-registry@0.73.1":
```

Comment:
It also looks like we introduced dupes of `@react-native/assets-registry` and other packages.